hermes/src/api/ws.rs

```diff
@@ -170,6 +160,8 @@ impl Subscriber {
                     "Config missing, price feed list was poisoned during iteration."
                 ))?;
 
+            // `sender.feed` sends a message to the client but does not flush it, so we can send
```

Comment:
Probably "buffers a message to the client" to be explicit.

Reply:
Thanks!